test/internal/modules/charge-information/lib/actions.js

```diff
@@ -81,14 +81,15 @@ experiment('internal/modules/charge-information/lib/actions', () => {
 
     test('does not restart the flow because there are charge elements', () => {
       const formValues = { startDate: 'today' };
-      request.pre.draftChargeInformation.chargeElements = [{ scheme: 'alcs' }];
+      request.pre.draftChargeInformation.chargeElements = [{ scheme: 'sroc' }];
```

Comment:
confirm this should change from alcs to sroc ? the service mr scheme is alcs. 

Ignore if you are happy.